app/assets/javascripts/standardize_time.js

```diff
@@ -0,0 +1,27 @@
+var timeStamp = $(".timestamp").text();
+var newDate = [];
+
+var standardizeTime = function() {
+  var toArr = function() {
+    return timeArr = timeStamp.split("  ");
```

Comment:
Did you mean to return a conditional instead of an assignment?<br>'timeArr' is not defined.